src/main/java/duke/util/List.java

```diff
@@ -0,0 +1,149 @@
+package duke.util;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+public class List {
+    //List that holds all the Tasks.
+    private final ArrayList<Task> list;
+    //Storage to load the list and write to list,
```

Comment:
I believe there is a typo here, it should be a fullstop intead a comma at the end of the sentence.:smile:

Reply:
nice catch haha thanks